Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Dockerfile to build an optimized version of the Compass server.
+#
+# To run the production image locally, create a .env.production file with the
+# appropriate env vars (a copy of .env.local works - see NOTE below).
+#
+# Command to build and run a Docker image:
+#  % docker build . -t compass && docker run -p=3000:3000 --name=compass --rm --env-file=.env.production compass
+#
+# NOTE: If you have the references to `localhost` in the .env file for other
+# services like postgres, replace `localhost` with the local IP address, e.g.
+# using `% ipconfig getifaddr en0`. This is needed as the docker image run in
+# its own VM and `localhost` resolves to to is own VM, not the host.
+
+# Start with the latest Node.js LTS release
+FROM --platform=linux/amd64 node:18-bullseye-slim
```

Comment:
does the arch need to be specified? makes it slower for people using arm machines

Reply:
Unfortunately yes - AWS needs amd64 images. 

Do you envision using the prod images locally on a regular basis or only for debugging the image? If the latter, the slowness might be tolerable. If the former, we can look into creating multi-platform images.

Reply:
Another option to build the image for arm is to specify the --platform in the CLI, e.g. `docker build --platform linux/arm64 .`

Reply:
if this is omitted, it will default to whatever the host machine is--I would recommend not specifying it here and passing it as `docker build --platform=linux/amd64` if building for x86 on ARM.

Reply:
AWS has ARM instances available, often for cheaper than x86; so defaulting to x86 doesn't really make sense to me. But 🤷

Reply:
FYI if you remove the platform flag you can use `docker buildx` to create and push multi architecture images to ECR in one step that contains both x86 and ARM layers, so that you can use either platform on AWS.